project.h

```diff
@@ -92,7 +92,14 @@
 #define DEBUG_DEVICE_IO
 
 #define RT_PRIO_UART    39
-#define RT_PRIO_BRIDGE  38
+/*
+ * The priority has to be at least 54 (same as SPI thread), otherwise lots
+ * of UART reception errors are observed. The reason for this effect is not
+ * yet completely clear, presumably the UART has to switch as quick as
```

Comment:
quick*ly*

Reply:
ah, sorry I just realized that you referred to _another_ typo in the comment section of the source code (I already fixed the same typo in the commit message)